### ISSUE TITLE
Attack Animation Mods that use DAR

### ISSUE THREAD

**Issue by MhiaSummers** [tiltedphoques/Mod-Compatibility#62]
# Any attack animation mods that use DAR to change/add animations for combat (attacking)

#### Download link
https://www.nexusmods.com/skyrimspecialedition/mods/53174 (DAR version of Elder Souls - The Collection)
or any other mods that use DAR to add/change attack animations

#### Dependencies
  - Nemesis
  - DAR

# Compatibility
> Player side sees the animation attack mod of your choice properly but any other players sees you doing different attacks from your DAR directory

# Additional Information
> Did some further testing and found out what the cause is, so DAR and many attack animation mods that use it implement this like looking for the player form id 00000007/00000014 so that way only the player character uses the animations and any npc uses what you set them to use, since other players are treated as npc's in your game they are using either default vanilla animation or what you have DAR set as folders for npc attacks. Little video showing this // https://www.youtube.com/watch?v=arJd7p0OY-o // (tested with ABR_AE - Attack Behavior Revamp, and I disabled player replacers so it would show vanilla attacks and other player will be using animations from npc files since their form id is not player form id)

> Player attack animations work well just showing differently for other player, no crashes

**Comment by Grimmspecter** [comment]
Presumably mods that add the animation to both NPCs and and player should be fine then? Have you tried animated armory?

**Comment by MhiaSummers** [comment]
> Presumably mods that add the animation to both NPCs and and player should be fine then? Have you tried animated armory?

As long as the animation mods you are using are replacers and not something managed by DAR it's all fine.

**Comment by SketchFoxsky** [comment]
We found out that any animations derived from Nemisis and FNIS will not work. DAR is the only animation replacer that seems to function properly in STR. "Player only" will be local.

**Comment by MhiaSummers** [comment]
> We found out that any animations derived from Nemisis and FNIS will not work. DAR is the only animation replacer that seems to function properly in STR. "Player only" will be local.

exactly, in case in the future nemesis and FNIS will work, this is going to be a problem, since Skysa/MCO and ABR has their own player only attacks and NPC only attacks.
